sgx/urts/src/eprintln.rs

```diff
@@ -3,18 +3,26 @@
 // Mobilenode-side support for sgx_debug crate impl
 //
 // Allows dev-build debugging messages similar to eprintln!
+// from within the enclave.
+//
+// We print them to slog because that is our logging infrastructure
+//
+// Note: This is deprecated and the eprintln should go away, the logger
+// API is what you should use.
 
+use mc_common::logger::global_log;
 use std::str;
 
 #[no_mangle]
 pub extern "C" fn eprintln_message(msg: *const u8, msg_len: usize) {
     let msg_bytes = unsafe { std::slice::from_raw_parts(msg, msg_len) };
 
     match str::from_utf8(msg_bytes) {
-        Ok(v) => eprintln!("Enclave log: {}", v),
-        Err(e) => eprintln!(
-            "Enclave log message contained invalid utf8:\n{}\n{:?}",
-            e, msg_bytes
+        Ok(v) => global_log::info!("Enclave eprintln: {}", v),
```

Comment:
What's the impetus for using `global_log` instead of `println`? Doesn't `println` already get redirected to slog anyways / isn't `println` basically already a global log of sorts? (I don't think moving to `global_log` is wrong, I'm just wondering what the motivation is)

Reply:
the impetus is that right now, when an enclave in the discovery container does `eprintln`, the logs for the container dont show me what server it even was, while `global_log` does. so from practical point of view it makes it less confusing

additionally I think we should treat those logs as part of the server log and make sure they follow the same path as everything else.

fwiw i think we should kill the eprintln thing now that we have `log::info!` working in enclave